renovate/default.json

```
@@ -77,6 +79,7 @@
     },
     {
       "groupName": "node",
+      "matchDatasources": ["docker", "node-version"],
       "matchPackageNames": ["@types/node", "node", "cimg/node", "actions/setup-node"],
       "enabled": false
```

I believe we could mandate a major instead of disabling?

```suggestion
      "allowedVersions": "22"
```

---

Before that, we'll have to make sure all repos actually are using this version

---

If not we can also create presets for each node version we want to support, so on repo side they can do 

```
{
  "extends": ["github>mui/mui-public//renovate/default", "github>mui/mui-public//renovate/node-20"]
}
```